portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function getServerQuotes() {
+  console.log("Has entered the getServerQuotes function");
+  fetch("/data").then(response => response.json()).then((comments) => {
+    let info = document.getElementById("comments");
+    info.innerHTML = '';
+    comments.forEach(element => {
+      info.appendChild( createListElement(
+        'Name: ' + element.fname
+        + " " + element.lname
```

Comment:
Let's be consistent with out use of single and double quotes. May be can change all of them to be single quotes?